### PR TITLE
adding GlobalizationExtensions APIs

### PR DESCRIPTION
**Pull request by mairaw**
@krwq, @tarekgh can one of you review the docs for the GlobalizationExtensions class and its APIs?

[Internal review URL](https://review.docs.microsoft.com/en-us/dotnet/api/system.globalization.globalizationextensions?view=netcore-2.0&branch=pr-en-us-3263)

### CODE REVIEW

File: xml/System.Globalization/GlobalizationExtensions.xml
Comment (tarekgh) [inline xml/System.Globalization/GlobalizationExtensions.xml]: >s either the enumeration value  <see cref="F:System.Globalization.CompareOptions.Ordinal" /> [](start = 99, length = 92)

The options can also be CompareOptions.OrdinalIgnoreCase

**Comment by tarekgh** [comment]
Left a small comment. otherwise LGTM.

**Review by BillWagner — APPROVED** [review]
This LGTM.

Once you have the product team approval, :shipit:

**Comment by mairaw** [comment]
Thanks @tarekgh. I've addressed your comment.

**Review by tarekgh — APPROVED** [review]

**Comment by rpetrusha** [comment]
The build completed but its status isn't reported here. Closing and reopening to begin new build.

**Pull request merged**
